lib/registration.pm

```diff
@@ -75,9 +75,10 @@ sub is_module {
 sub accept_addons_license {
     my (@scc_addons) = @_;
 
-    my @addons_with_license = qw(ha geo we live rt idu ids lgm wsm hpcm ses);
-    # Development tools do not have license in SLE 15
-    push(@addons_with_license, 'sdk') unless sle_version_at_least('15');
+    my @addons_with_license = qw(ha geo we live rt idu ids lgm hpcm ses);
+    # Development tools and Web-Scripting modules do not have license in SLE 15
+    push(@addons_with_license, 'sdk') unless is_sle('15+');
```

Comment:
just for info (no change needed) - using `if is_sle('<15')` is sometimes safer in case you share code with opensuse because it also checks if distri=sle